frontend/src/services/fetch/fetchers/opportunityFetcher.ts

```diff
@@ -0,0 +1,11 @@
+import { OpportunityApiResponse } from "src/types/opportunity/opportunityResponseTypes";
```

Comment:
since the bare fetcher no longer returns the json result, we'll create a basic wrapper here.